verdict/src/main.rs

```diff
@@ -0,0 +1,123 @@
+use clap::{App, Arg};
+use serde_json::Value;
+use std::fs::File;
+use std::io::prelude::*;
+
+use configureProvider::configure_provider_service_client::ConfigureProviderServiceClient;
+use configureProvider::{ExportPolicyRequest, ExportPolicyResponse};
+use configureProvider::{SetPolicyRequest, SetPolicyResponse};
+use configureProvider::{SetRawPolicyRequest, SetRawPolicyResponse};
+
+pub mod configureProvider {
+    tonic::include_proto!("configureprovider");
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let matches = App::new("verdict")
+        .version("0.1")
+        .author("Inclavare-Containers Team")
+        .arg(
+            Arg::with_name("set_policy")
+                .short("s")
+                .long("set_policy")
+                .value_name("POLICY_NAME")
+                .value_name("FILE_REFERENCE")
+                .help("Generate a policy file named <POLICY_NAME>, according to the contents in <FILE_REFERENCE>.")
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name("set_raw_policy")
+                .short("r")
+                .long("set_raw_policy")
+                .value_name("POLICY_NAME")
+                .value_name("FILE")
+                .help("Write the contents of <FILE> into the policy file named <POLICY_NAME>.")
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name("export_policy")
+                .short("e")
+                .long("export_policy")
+                .value_name("POLICY_NAME")
+                .help("Export the contents of the policy file named <POLICY_NAME>.")
+                .takes_value(true),
+        )
+        .get_matches();
+
+    let mut client = ConfigureProviderServiceClient::connect("http://[::1]:60000").await?;
+
+    // set_policy
+    if matches.is_present("set_policy") {
+        let vals: Vec<&str> = matches.values_of("set_policy").unwrap().collect();
```

Comment:
These handle logic codes should be put into a separate function because we may add others interfaces in future which will make this main() function too large.